tests/src/Extensions/TransformExtension.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\nexteuropa\Extensions\TransformExtension.
+ */
+
+namespace Drupal\nexteuropa\Extensions;
+
+use Behat\Testwork\ServiceContainer\Extension as ExtensionInterface;
+use Behat\Behat\Transformation\ServiceContainer\TransformationExtension;
+use Behat\Testwork\ServiceContainer\ExtensionManager;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * Class TransformExtension.
+ *
+ * @package Drupal\nexteuropa\Extensions
+ */
+class TransformExtension implements ExtensionInterface {
+
+  /**
+   * Extension configuration ID.
+   */
+  const NEPT_TRANSFORM_ID = 'nept_transform';
+
+  /**
+   * Implement getConfigKey() method.
+   *
+   * @inheritDoc
+   */
+  public function getConfigKey() {
+    return self::NEPT_TRANSFORM_ID;
+  }
+
+  /**
+   * Implement initialize() method.
+   *
+   * @inheritDoc
+   */
+  public function initialize(ExtensionManager $extension_manager) {
+
+  }
+
+  /**
+   * Implement configure() method.
+   *
+   * @inheritDoc
+   */
```

Comment:
I will change but do not use a Drupal reference in project that does not concern Drupal directly.